### DISCUSSION TITLE
Some questions i have about the project.

### DISCUSSION THREAD

**Discussion by Felipellira** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#288] [Q&A] [answered]

**Comment by Felipellira** [comment]
My question has already been answered elsewhere

**Comment by joelwurtz** [comment, marked as answer]
I was going to respond, will still do it if anyone has the same questions

> Why do you use qbcore and soz-core, considering that the two have similar purposes?

qbcore is a legacy dependency it will be removed totally at some point but we migrate everything to soz core step by step so it's still necessary

> Do you have a specific reason for using decorators? For example not storing commands in an array, I feel that decorators only make the project more complex, is there any functionality or performance problem when we don't use decorators?

This is mainly for DX (developer experience) it allow to remove a lot of boilerplate, once it's loaded everything is stored / registered in array or using natives from FiveM. There is no performance penalty by using this, it just allow us to better organize ourself

> Why transport data using reflect-metadata?

Don't know what you mean about transport data, but reflect-metadata is necessary for using decorators
